doc/release/1.13.0-notes.rst

```diff
@@ -22,6 +22,8 @@ Dropped Support
 Deprecations
 ============
 
+* ``in1d`` is now deprecated in favor of ``isin``.
```

Comment:
This may be my fault: I may have said something like to suggest that we should deprecate in1d, but I was being sloppy with my language. I don't think we can (or should) ever raise a DeprecationWarning. `in1d` works just fine, albeit not with as nice an API as `isin`, and is used all over the place, so there is absolutely no good reason to remove it.

What we can and should do is *discourage* using `in1d` in favor of `isin` for new code. Instead of saying `in1d` is deprecated, its docstring should say something like: "We recommend using `isin` instead of `in1d` for new code".